notebooks/106-speech-recognition-quantize-wav2vec2/106-speech-recognition-quantize-wav2vec2.ipynb

```diff
@@ -0,0 +1,631 @@
+    "        res_string = ' '.join(res_string.split(' '))\n",
+    "        res_string = res_string.lower()\n",
+    "        return res_string\n",
+    "\n",
+    "    def _editdistance_eval(self, source, target):\n",
+    "        n, m = len(source), len(target)\n",
+    "\n",
+    "        distance = np.zeros((n+1, m+1), dtype=int)\n",
+    "        distance[:, 0] = np.arange(0, n+1)\n",
+    "        distance[0, :] = np.arange(0, m+1)\n",
+    "\n",
+    "        for i in range(1, n + 1):\n",
+    "            for j in range(1, m + 1):\n",
+    "                cost = 0 if source[i - 1] == target[j - 1] else 1\n",
+    "\n",
+    "                distance[i][j] = min(distance[i - 1][j] + 1,\n",
+    "                                    distance[i][j - 1] + 1,\n",
+    "                                    distance[i - 1][j - 1] + cost)\n",
+    "        return distance[n][m]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9bbbca4a",
+   "metadata": {
+    "id": "CclWk-fVd9Wi"
+   },
+   "source": [
+    "## Run Quantization Pipeline\n",
+    "Here we define a configuration for our quantization pipeline and run it. Please note that we use built-in `IEEngine` implementation of `Engine` interface from the POT API for model inference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16457bd4",
+   "metadata": {
+    "id": "PiAvrwo0tr6Z",
+    "tags": [],
+    "test_replace": {
+     "\"stat_subset_size\": 300,": "\"stat_subset_size\": 3,"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "model_config = Dict({\"model_name\": \"wav2vec2_base\", \"model\": ir_model_xml, \"weights\": ir_model_bin})\n",
+    "engine_config = Dict({\"device\": \"CPU\"})\n",
+    "dataset_config = {\n",
+    "    \"data_source\": os.path.join(DATA_DIR, \"LibriSpeech/test-clean\"),\n",
+    "}\n",
+    "algorithms = [\n",
+    "    {\n",
+    "        \"name\": \"DefaultQuantization\",\n",
+    "        \"params\": {\n",
+    "            \"target_device\": \"ANY\",\n",
+    "            \"model_type\": \"transformer\",\n",
+    "            \"preset\": \"performance\",\n",
+    "            \"stat_subset_size\": 300,\n",
+    "            \"activations\": {\n",
+    "                \"range_estimator\": {\n",
+    "                    \"min\": {\n",
+    "                        \"aggregator\": \"min\",\n",
+    "                        \"type\": \"min\"\n",
+    "                    },\n",
+    "                    \"max\": {\n",
+    "                        \"aggregator\": \"mean\",\n",
+    "                        \"type\": \"quantile\",\n",
+    "                        \"outlier_prob\": 0.0001 \n",
+    "                    }\n",
+    "                }\n",
+    "            },\n",
+    "            \"ignored\": {\n",
+    "                \"scope\": [\"214\", \"Add_843\", \"Conv_19\", \"Mul_9\", \"Conv_1\", \"MatMul_620\", \"MatMul_841\",\n",
```

Comment:
Where do layers like `Add_843`, `Conv_19`, etc. come from? The FP IR doesn't contain these layers. Please, verify that all layers from the ignored scope exist in the model. Also, `214` in the ignored scope is a WA for the known runtime issue. Please, note, that it should be removed in the future.

Reply:
You're right, ignored layers block is useless. I've just submitted fix. Only '214' left as a WA for known issue.